WordPress/Classes/Utility/App Configuration/BuildSettings.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+public struct BuildSettings: Sendable {
+    public var itunesAppID: String
+    public var pushNotificationAppID: String
+
+    public init(
+        itunesAppID: String,
+        pushNotificationAppID: String
+    ) {
+        self.itunesAppID = itunesAppID
+        self.pushNotificationAppID = pushNotificationAppID
+    }
+
+    static var current: BuildSettings {
+        guard let value = _current else {
+            fatalError("configuration not registered")
+        }
+        return value
+    }
+
+    private static var _current: BuildSettings?
```

Comment:
```suggestion
    static var current: BuildSettings!
```

WDYT?

Reply:
It's now configured before any other code gets a chance to run, so it's effectively non-optional. Implicitly unwrapped optional doesn't provide the same guarantee. It's mainly used for importing Objective-C APIs that don't explicitly specify optionality, so I'd prefer to stick with a non-optional.

```swift
// Here settings becomes optional `BuildSettings?` which isn't ideal
let settings = BuildSettings.shared
```